pallets/msa/src/runtime-api/src/lib.rs

```diff
@@ -29,7 +29,7 @@ sp_api::decl_runtime_apis! {
 	/// - MUST be incremented if anything changes
 	/// - Also update in js/api-augment
 	/// - See: https://paritytech.github.io/polkadot/doc/polkadot_primitives/runtime_api/index.html
-	#[api_version(1)]
+	#[api_version(2)]
```

Comment:
I think we can leave this v1 and just tag the `get_all_granted_delegations_by_msa_id` with `#[api_version(2)]`

That way anyone using v1 doesn't have to change because `get_granted_schemas_by_msa_id` doesn't change.

See the example here: https://paritytech.github.io/polkadot-sdk/master/polkadot_primitives/runtime_api/index.html#runtime-api-versioning-crash-course

Reply:
Unfortunately it doesn't work as documentation suggests. There might be some kind of bug with `impl_runtime_apis` macro implementation that doesn't compile when I add `#[api_version(2)]` on top of the new method. 
This generate two version for RuntimeApi but it doesn't let me to implement the V2 due to different compile errors. I tried a bunch of things but none of them seemed to work. Please feel free to take a stab at it but as of now we have to keep the implementation as is.